integration/samples/secondary/specs/es2016-api.ts

```diff
@@ -1,12 +1,14 @@
 import { expect } from 'chai';
 import * as fs from 'fs';
 import * as path from 'path';
-import * as API from '../dist/esm2015/secondary-lib.js';
-import * as SECONDARY from '../dist/esm2015/sub-module.js';
 
 describe(`@sample/secondary-lib`, () => {
 
   describe(`secondary-lib.js`, () => {
+    let API;
+    before(() => {
+      API = require('../dist/esm2015/secondary-lib.js');
+    });
```

Comment:
Prevents TypeScript compilation error for spec suite. Fails `before` hook in test run, if bundle files does not exist.